bml/lib/data_summary.py

```diff
@@ -33,29 +42,65 @@ def data_summary(data):
     std_dev = "{:.4f}".format(numpy.std(data)).ljust(10)
     avg = "{:.4f}".format(numpy.mean(data)).ljust(10)
     median = "{:.4f}".format(numpy.median(data)).ljust(10)
-    return("Std_dev: " + std_dev + " Avg: " + avg + " Med: " + median)
+    summary=[avg,std_dev,median]
+    return(summary)
 
 
 # Used for an easy way to look at run data
 def print_run_details(config, es_backend, uuid):
     brun = browbeat_run(es_backend, uuid, caching=True)
     output_string = ""
     osp_version = ""
-    padding = longest_test_name(config)
+    padding = longest_test_name(config) #gets the test with highest run time
+    '''
+    conn=sqlite3.connect('database.sqlite')
+    c=conn.cursor()
+    '''
     for test_type in config['tests']:
         test_name = test_type['test']
         data = []
         for test_run in brun.get_tests(test_search=test_name):
             if 'pipeline' in test_run.dlrn_hash or \
                'trunk' in test_run.dlrn_hash:
                 return False
-            elif 'HA' not in test_run.cloud_name and \
-                 'ha' not in test_run.cloud_name:
+            if len(str(test_run.dlrn_hash))<=45:
                 return False
             data.extend(test_run.raw)
             osp_version = test_run.version
+
+        average_runtime=data_summary(data)[0]
         output_string += test_name.ljust(padding) + \
-            " " + data_summary(data) + "\n"
+            " " + str(average_runtime)  #typecasting data_summary(data) to string because if it returns false it's going to throw an error
+        if float(average_runtime)>0.0 and test_run.errortype=="result" and test_name!="nova.boot_server":
+            if str(test_name) in config['test_with_scenario_list']:
+                test_name=str(test_run.scenario_name)+"."+str(test_name)
+            output_prediction=classify_value(config,average_runtime,test_name,osp_version)
+            if str(output_prediction[0])=="1":
+                print("ALERT!!!!")
+                print (uuid,test_name,predictors[2])
+                exit(1)
+
+            #print("EEEEEEEEEEENNNNNNNNNNDDDDDD")
+            output_string=output_string+ str(output_prediction) + "\n"
+            #c.execute('INSERT INTO scenario_test VALUES (?,?,?,?,?)',(str(uuid),str(osp_version),str(test_run.scenario_name),str(test_name),average_runtime))
+            '''
+            if test_run.cloud_name in config['clouds']:
+                c.execute('INSERT INTO avg_values2 VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)',\
+                          (str(uuid),str(test_run.timestamp),str(osp_version),str(test_run.cloud_name),\
+                           str(test_run.os_name),str(test_run.kernel),test_run.no_computes,\
```

Comment:
This part is actually commented but fixed the variable nomenclature and also ensured that it now displays the std_dev.